nxdrive/metrics.py

```diff
@@ -0,0 +1,40 @@
+from logging import getLogger
+from queue import Empty, Queue
+from typing import Any, Dict
+
+from .engine.workers import Worker
+from .exceptions import ThreadInterrupt
+
+log = getLogger(__name__)
+
+
+class DriveMetrics(Worker):
+    def __init__(self, remote) -> None:
+        super().__init__("DriveMetrics")
+
+        self._remote = remote
+        self._metrics_queue = Queue()
+        self._stop = False
+        self.thread.started.connect(self.run)
+
+    def _execute(self) -> None:
+        try:
+            while True:
+                try:
+                    metrics = self._metrics_queue.get(timeout=1)
+                    print(f"Sending {metrics}")
+                    if not metrics:
+                        continue
+                    self._remote.client.request(
+                        "GET", "/site/api/v1/me", headers=metrics
```

Comment:
Move `/site/api/v1/me` to a constant at the top of the file: `_ENDPOINT`.

Reply:
I think we should also set a timeout. Could you check in logs what is the timeout for such request?